src/index.ts

```diff
@@ -69,14 +69,14 @@ program
 
       if (raw) {
         dir(processes, { depth: null })
-      } else {
+      } else{
```

Comment:
This is concerning there's a whitespace issue here that the linter should have caught but didn't. @n3op2 can you have a look and see if you can work out what's wrong in the linter config